pheps/phep-0001.md

```diff
@@ -0,0 +1,338 @@
+
+PHEPs are the primary mechanism for proposing standards, collecting community input, and documenting the design decisions underlying PyHC standards and recommendations. The PHEP process develops and documents consensus within the community. The PHEP author is responsible for building this consensus and documenting dissenting opinions.
+
+Because PHEPs are maintained as text files in a versioned repository, their revision history is the historical record of the feature proposal. This record is available by the normal git commands for retrieving older revisions and can be browsed on GitHub.
+
+# PHEP Motivation
+<a name="phep-motivation"></a>
+The [PyHC standards](https://doi.org/10.5281/zenodo.2529131) have not been updated since their initial approval in 2018, and there is no established process for updating them. PyHC discussions often involve what packages "should" do, or "should be required" to do. For standards to be effective, they need to be seen as legitimate by those they may be imposed on.
+
+A mechanism is needed to capture the sense of the community and allow it to progress while recognizing that the community is broad, diffuse, and diverse, and not all stakeholders are present for all conversations.
+
+# PHEP Audience
+<a name="phep-audience"></a>
+The typical audience for PHEPs is the developers and user communities of PyHC packages. Other parts of the PyHC community (hereafter "community") may also use the process (particularly for Informational PHEPs) to document conventions and to manage complex design coordination problems that require collaboration across multiple projects.
+
+This PHEP refers to "PyHC leadership": individuals or groups with statutory authority over the PyHC project. This may be the Principal Investigator or other individuals or groups appropriately designated in the future, such as a steering committee.
+
+# PHEP Types
+<a name="phep-type"></a>
+There are three types of PHEP:
+
+1. A **Standards** Track PHEP describes a new or modified standard for PyHC packages.
+
+2. An **Informational** PHEP describes a design issue, or provides general guidelines or information to the community, but does not propose a standard. Informational PHEPs do not necessarily represent a community recommendation, so users and implementers are free to ignore Informational PHEPs or follow their advice.
+
+3. A **Process** PHEP describes a process surrounding PyHC, or proposes a change to a process. Process PHEPs are like Standards Track PHEPs but apply to areas other than PyHC packages. They may propose an implementation; they often require community consensus; unlike Informational PHEPs, they are more than recommendations, and users are typically not free to ignore them. Examples include procedures, guidelines, and changes to the decision-making process.
```

Comment:
Ooof, I'm inclined to move the reference to "implementation" here. In the context of Standards PHEP it would be "a package we assert is compliant"; in the context of Informational it would be "here is the best practice." But _process_ PHEPs are unlikely to have an "implementation" unless it's automation support for the process.

And I think all PHEPs require consensus, as written. So maybe this should be:
"~They may propose an implementation; they often require community consensus;~ Unlike Informational PHEPs, they are more than recommendations, and users are typically not free to ignore them."

Reply:
[New wording](https://github.com/jtniehof/standards/blob/e335aacd1346f0876f5a54d9d617bf3c11ec2e32/pheps/phep-0001.md?plain=1#L41)